SR2025/2024-09-26-kickstart-details.md

```diff
@@ -0,0 +1,56 @@
+---
+to: All currently signed-up teams
+subject: SR2025 Kickstart details
+---
+
+Hi!
+
+Kickstart is on October 12th, only a couple weeks away! So it's time to give you a bit more info about the event.
+
+Kickstart is the big kick-off event for the competition. We'll firstly be announcing the game for which you'll be building your robots, alongside some additional challenges to complete along the way. Finally, we'll present the 'Microgames', a series of tasks to get acquainted with your new kits. Our blueshirts (our lovely volunteers) will be available to mentor your teams throughout.
+
+If you can't attend physically, it's still a good idea to get your team together to attend virtually. We'll be [livestreaming][livestream] the game presentation from 11am and will release the microgames once the presentation is over. We'll also have volunteers available to help out with the Microgames on [Discord][discord] both on the day and for the rest of the competition year.
+
+Please do [let us know][teams-contact] if your plans change regarding your physical attendance at Kickstart.
+
+We'll keep the [event page](https://studentrobotics.org/events/sr2025/kickstart/) up-to-date with the latest info, but for your convenience, the important information is below.
+
+## Directions
+
+Kickstart is being held at the University of Southampton at:
+
+    Building 46,
+    University Road,
+    Southampton,
+    SO17 1BJ
+
+For public transport or driving instructions, see the University of Southampton’s information on ['getting to our campuses'](https://www.southampton.ac.uk/student-life/campuses/highfield) (click the ‘Campus map and directions’ tab).
```

Comment:
nit: maybe quotes outside the link? Or drop them entirely? (not sure they add anything as I'm not sure what we're quoting)
```suggestion
For public transport or driving instructions, see the University of Southampton’s information on '[getting to our campuses](https://www.southampton.ac.uk/student-life/campuses/highfield)' (click the ‘Campus map and directions’ tab).
```
```suggestion
For public transport or driving instructions, see the University of Southampton’s information on [getting to our campuses](https://www.southampton.ac.uk/student-life/campuses/highfield) (click the ‘Campus map and directions’ tab).
```